docs/examples/screenly/README.md

```diff
@@ -0,0 +1,130 @@
+
+![screenly](https://github.com/WoTTsecurity/agent/blob/master/docs/examples/screenly/screenly-setup.jpg)
+
+Then navigate to the [Address](https://www.screenly.io/wifi) shown and login with your network details. This will grant you access to the Screenly management page. 
+
+Now we need to secure access to your Screenly device. If you haven't installed the WoTT Agent already, install [it](https://github.com/WoTTsecurity/agent).
+
+To do this on the Screenly device, you need to access the terminal through `CTRL` + `ALT` + `F1`. To return back to the GUI, it is `CTRL` + `ALT` + `F2`. 
+Once here, follow the WoTT agent installation as you would on any other device. 
+
+**Optional:** If you want to avoid using the terminal directly on your Screenly device in the future, enable SSH through the `sudo raspi-config` command. It is also reccommended that you change your User and Password from the default `pi` and `raspberry` (this will also improve your WoTT security score!) if you are going to do this.
+
+You will now need to register the Pi on the WoTT dashboard. 
+
+
+## Downloading WoTT credentials on Screenly OSE
+
+Login to the WoTT Dash and navigate to 'Claim Device.' If you have installed the WoTT agent recently, the relevant information should be displayed on your terminal screen. If not, use the following commands:
+
+``` 
+$ sudo wott-agent whoami
+$ sudo wott-aget claim-token
+```
+
+This will give you your Device ID and the token value to claim your device. 
+
+Your Raspberry Pi should now be registered. Navigate to your Dashboard and select the Pi:
+
+![pi](https://github.com/WoTTsecurity/agent/blob/master/docs/examples/screenly/rasbpi.png)
+
+On the overview page, add a tag. This will be important for the credentials. This tag can be whatever you want, however we suggest something like `screenly-pi` or `pi` or a variant of that form:
+
+![tag](https://github.com/WoTTsecurity/agent/blob/master/docs/examples/screenly/tag.png)
+
+Now navigate to credentials and add a new credential with the following details:
+
+```
+Name = screenly
+Key = login
+Value = username:password
+Owner = user
+Tags = screenly-pi
+```
+Where the Owner is a valid Linux username, specifically the Linux user running Screenly (so by default on a Raspberry Pi, `pi`) and Key must be `login` (or the credentials won't be read). Value is the actual username and password of the login denoted by the single field `username:password`. You can change this value to match your own criteria, but it's fine for this example to leave it as is. 
+
+You can change the Name value of `screenly` if you wish, but note that this is how WoTT will call the credential information, so if you do change this name, be aware to use the correct credential name later on. Make sure as well that the Tags match whatever Tag you assigned the Pi earlier.
+
+You now need to set up the screenly configuration file to call the WoTT credentials. This file is in a hidden folder, so to access it do the following:
```

Comment:
You can skip the text edit part as I've already added that in upstream.

Reply:
does this assume that the name of the file is 'screenly' ?

Reply:
yep.